queries/nu/highlights.scm

```diff
@@ -0,0 +1,396 @@
+; ---
+; keywords
+[
+  "def"
+  "alias"
+  "export-env"
+  "export"
+  "extern"
+  "module"
+  "let"
+  "let-env"
+  "mut"
+  "const"
+  "hide-env"
+  "source"
+  "source-env"
+  "overlay"
+  "register"
+  "loop"
+  "while"
+  "error"
+  "do"
+  "if"
+  "else"
+  "try"
+  "catch"
+  "match"
+  "break"
+  "continue"
+  "return"
+] @keyword
+
+(hide_mod
+  "hide" @keyword)
+
+(decl_use
+  "use" @keyword)
+
+(ctrl_for
+  "for" @keyword
+  "in" @keyword)
+
+(overlay_list
+  "list" @keyword.import)
+
+(overlay_hide
+  "hide" @keyword.import)
+
+(overlay_new
+  "new" @keyword.import)
+
+(overlay_use
+  "use" @keyword.import
+  "as" @keyword)
+
+(ctrl_error
+  "make" @keyword.import)
+
+; ---
+; literals
+(val_number) @number
+
+(val_duration
+  unit: _ @variable.parameter)
+
+(val_filesize
+  unit: _ @variable.parameter)
+
+(val_binary
+  [
+    "0b"
+    "0o"
+    "0x"
+  ] @number
+  "[" @punctuation.bracket
+  digit: [
+    "," @punctuation.delimiter
```

Comment:
This is overridden by the query below (which should also be `@punctuation.delimiter` rather than `@punctuation.special`)